tests/baselines/reference/enumKeysQuotedAsObjectPropertiesInDeclarationEmit.js

```diff
@@ -52,7 +52,7 @@ export declare enum MouseButton {
     NO_BUTTON = 0
 }
 export declare const DOMMouseButton: {
-    '-1': MouseButton;
+    "-1": MouseButton;

```

Comment:
This is a bit of a strange one, because sometimes TS normalizes strings that come from values and sometimes it does not.  If you assign a string to a const, TS never preserved those as written and replaced the string delimiters. I preserved this behavior, and it is now applied to string property names:

```
const x = '1' // We get "1" in d.ts
const y = { '1': ''} // We get '1' in d.ts.
```

Reply:
I guess I don't know why this is a desirable change, if the behavior differs depending on where the string is used...

Reply:
I opened a PR that shows what the impact of preserving delimiters more widely would be: https://github.com/microsoft/TypeScript/pull/60729